openslides_backend/action/action.py

```diff
@@ -107,7 +107,13 @@ def perform(
             self.validate_instance(instance)
             # perform permission check not for internal actions
             if not internal:
-                self.check_permissions(instance)
+                try:
+                    self.check_permissions(instance)
+                except PermissionDenied as e:
+                    if e.message.startswith("Missing"):
```

Comment:
Why do you check the start of the error message? I think it should be added to all messages.

Reply:
No, it should not added to all messages. There are as example anonymous permission denied and special permission messages.

Reply:
Than use inheritance and create special child classes of `PermissionDenied`.

Reply:
Then use inheritance and create special child classes of `PermissionDenied`.

Reply:
Yes, that would have also been my next suggestion, didn't think about the possible other error causes. Maybe something like `MissingPermission`?

Reply:
Add `MissingPermission `exception and use it.